examples/experimental/h3-grid/webpack.config.js

```diff
@@ -36,7 +36,6 @@ const CONFIG = {
   },
 
   plugins: [
-    new webpack.EnvironmentPlugin(['MapboxAccessToken']),
```

Comment:
If you remove this, you also need to remove the previous require of webpack

Reply:
fixed: dbebf57